emails/management/commands/delete_old_reply_records.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime, timedelta, timezone
+
+from django.core.management.base import BaseCommand
+
+from ...models import Reply
+
+
+class Command(BaseCommand):
+    help = 'Deletes all Reply objects older than 3 months.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('days_old', nargs=1, type=int)
+
+    def handle(self, *args, **options):
+        delete_date = (
+            datetime.now(timezone.utc) - timedelta(options['days_old'][0])
+        )
+        replies_to_delete = (
+            Reply.objects.filter(created_at__lt=delete_date)
+            .only('id')
+        )
+        print(f'Deleting {len(replies_to_delete)} reply records')
+        replies_to_delete.delete()
```

Comment:
Consider using `atomic` for possible race condition and/or adding logging here to surface errors to delete replies. If it's not deleted immediately it will be deleted the next time the script will run, likely the day after.

Reply:
`delete()` should run a bulk (i.e., single) `DELETE` statement, so I don't think `atomic` is required, but I'll include it for extra defensive coding.

Any exception should surface up into Sentry as usual, IIUC. @moz-astults - exceptions & errors during cron jobs still pop up in Sentry, right?

Reply:
@groovecoder I'm not sure if you need to wrap the exception with the sentry_sdk [like this](https://github.com/mozilla/fx-private-relay/blob/8d5194841a971a98e376dc836dee049b51827aa7/privaterelay/views.py#L242) or not, but if there's some way it handles exceptions globally then I think it should work. The `SENTRY_DNS` env var will be in place on the cron job.

Reply:
Yup, we call `sentry_sdk.init` from `privaterelay.settings` module which is run for management commands like this one. So Sentry should receive any errors during this command.